database_schema.sql

```diff
@@ -101,25 +114,34 @@ CREATE TABLE `errors` (
 CREATE TABLE `languages` (
   `id` int(11) UNSIGNED NOT NULL,
   `language` varchar(25) NOT NULL,
+  `digit` int(1) UNSIGNED NOT NULL,
```

Comment:
I think digit is too ambiguous as well. Perhaps dial_number? I'm actually having a hard thinking of a good rename all I know is that "digit" hard to understand.